site/docs/v1/tech/samlv2/index.adoc

```diff
@@ -8,61 +8,103 @@ description: An overview of the SAML identity provider capabilities of FusionAut
 
-* Identity provider - the entity that is performing the authentication of a user. Essentially, this is the thing that is providing the login page.
-* Service provider - the entity that needs a user to login with an identity provider in order to provide some service. Essentially, this is the app the user wants to use.
+* Identity provider - the entity that is performing the authentication of a user. Essentially, this is the thing that is providing the login page. Also known as an IdP.
+* Service provider - the entity that needs a user to login with an identity provider in order to provide some service. Essentially, this is the app the user wants to use. Also known as an SP.
 
-As a concrete example, you might have an app called `Payroll Zen` that manages your companies payroll. This app requires that employees who have access to use it login in using your companies Active Directory. In this case, `Payroll Zen` is the service provider and Active Directory is the identity provider.
+As a concrete example, you might have an app called `Payroll Zen` that manages your company's payroll. This app requires that employees who have access to use it log in using the credentials stored in your company's Active Directory. `Payroll Zen` is therefore the service provider and Active Directory is the identity provider.
 
 include::docs/v1/tech/samlv2/_saml_idp_note.adoc[]
 
-It should also be noted that FusionAuth's SAML identity provider uses the OAuth workflow under the hoods. When a service provider starts a SAML workflow by sending a SAML request to FusionAuth, FusionAuth will forward the browser to its own OAuth UI. This is an important concept if you are interested in using FusionAuth's OAuth theme support to customize the look and feel of your SAML identity provider.
+FusionAuth's SAML identity provider uses the OAuth workflow under the hoods. When a service provider starts a SAML workflow by sending a SAML request to FusionAuth, FusionAuth will forward the browser to the hosted login pages. This is an important concept if you are interested in using FusionAuth's themes to customize the look and feel of your SAML identity provider user interface.
 
-== Configure SAML Identity Provider
+== Configure FusionAuth as a SAML Identity Provider
 
-In order to configure FusionAuth to act as a SAML identity provider, you need to specify the SAML configure for an Application. This will allow FusionAuth to present a SAML login page that will allow users to login for that application. In this case, FusionAuth will consume SAML requests from the application and return SAML responses to the application after the user logs in.
+In order to configure FusionAuth to act as a SAML identity provider, you need to enable and configure SAML for an Application. When properly configured, FusionAuth, the IdP, will consume SAML requests from an application, the SP, and, after a user logs in, return proper SAML responses.
```

Comment:
You could also use the words "Authentication request" and "Authentication response"